src/i18n/es.js

```diff
@@ -291,7 +291,7 @@ export const messages_es = {
   'editor.tablet.description': `En este momento está editando para dispositivos de tableta. Cambie para todas las opciones de estilo.`,
   'editor.text.label': `Texto`,
   'editor.text_align.label': `Alineación Texto`,
-  'editor.text_color.label': `Color Texto`,
+  'editor.text_color.label': `Color texto`,
```

Comment:
Según [lo hablado](https://makingsense.slack.com/archives/CLBEN4JAV/p1678994237063369?thread_ts=1678992624.890959&cid=CLBEN4JAV) con @GGossweiler:

```suggestion
  'editor.text_color.label': `Color del texto`,
```